src/mongocxx/test/client_side_encryption.cpp

```diff
@@ -571,29 +590,34 @@ void run_external_key_vault_test(bool with_external_key_vault) {
 TEST_CASE("External key vault", "[client_side_encryption]") {
     instance::current();
 
+    if (!mongocxx::test_util::should_run_client_side_encryption_test()) {
+        return;
+    }
+
     class client setup_client {
-        uri{}, test_util::add_test_server_api(),
+        uri{}, test_util::add_test_server_api(crypt_shared_opts()),
```

Comment:
Suggest removing `crypt_shared_opts()`. The `setup_client` is only used to check wire version. I do not expect automatic encryption is needed.